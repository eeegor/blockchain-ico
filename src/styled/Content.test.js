import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { Content } from '.';

describe('Content', () => {
	test('renders content', () => {
		const tree = renderer.create(<Content>Content</Content>).toJSON();
		expect(tree).toMatchSnapshot();
	});

	test('renders content with top offset', () => {
		const tree = renderer
			.create(<Content top={100}>Content</Content>)
			.toJSON();
		expect(tree).toMatchSnapshot();
		expect(tree).toHaveStyleRule('margin-top', '100px');
	});

	test('renders content with top offset', () => {
		const tree = renderer
			.create(<Content top={[100, 200]}>Content</Content>)
			.toJSON();
		expect(tree).toMatchSnapshot();
		expect(tree).toHaveStyleRule('margin-top', '100px');
	});
});
