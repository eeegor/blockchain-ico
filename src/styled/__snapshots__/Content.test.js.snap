// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content renders content 1`] = `
.c0 {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: no-wrap;
  -ms-flex-wrap: no-wrap;
  flex-wrap: no-wrap;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  overflow: scroll;
  overflowscrolling: touch;
  webkitoverflowscrolling: touch;
  -webkit-overflow-scrolling: touch;
  margin-top: 0;
  overflow: hidden;
  background: #F6F7FB;
  padding: 4rem 0;
  -webkit-overflow-scrolling: touch;
  z-index: 5000;
}

<div
  className="c0"
>
  Content
</div>
`;

exports[`Content renders content with top offset 1`] = `
.c0 {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: no-wrap;
  -ms-flex-wrap: no-wrap;
  flex-wrap: no-wrap;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  overflow: scroll;
  overflowscrolling: touch;
  webkitoverflowscrolling: touch;
  -webkit-overflow-scrolling: touch;
  margin-top: 0;
  overflow: hidden;
  background: #F6F7FB;
  padding: 4rem 0;
  -webkit-overflow-scrolling: touch;
  z-index: 5000;
  margin-top: 100px;
}

@media (min-width:767px) {
  .c0 {
    margin-top: undefinedpx;
  }
}

<div
  className="c0"
>
  Content
</div>
`;

exports[`Content renders content with top offset 2`] = `
.c0 {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: no-wrap;
  -ms-flex-wrap: no-wrap;
  flex-wrap: no-wrap;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  overflow: scroll;
  overflowscrolling: touch;
  webkitoverflowscrolling: touch;
  -webkit-overflow-scrolling: touch;
  margin-top: 0;
  overflow: hidden;
  background: #F6F7FB;
  padding: 4rem 0;
  -webkit-overflow-scrolling: touch;
  z-index: 5000;
  margin-top: 100px;
}

@media (min-width:767px) {
  .c0 {
    margin-top: 200px;
  }
}

<div
  className="c0"
>
  Content
</div>
`;
